### PR TITLE
feat: univ2 lp off-boarding

### PR DESCRIPTION
**Pull request by nduchak**
# Offboard UNI-A, UNIV2DAIETH-A, UNIV2WBTCETH-A, UNIV2UNIETH-A, and UNIV2WBTCDAI-A

## UNI-A
- `tip` to `0`
- `chop` to `0` 
- `mat` from `165` to `900` duration `30days`

## UNIV2DAIETH-A
- `tip` to `0`
- `chop` to `0`
- `mat` from `120` to `3100` duration `30days`
- 
## UNIV2WBTCETH-A
- `tip` to `0`
- `chop` to `0`
- `mat` from `145` to `5300` duration `30days`

## UNIV2UNIETH-A
- `tip` to `0`
- `chop` to `0`
- `mat` from `160` to `700` duration `30days`

## UNIV2WBTCDAI-A
- `tip` to `0`
- `chop` to `0`
- `mat` from `120` to `1000` duration `30days`

### CODE REVIEW

File: src/Goerli-DssSpell.sol [hunk 1]
```diff
@@ -36,6 +36,23 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     // Provides a descriptive tag for bot consumption
     string public constant override description = "Goerli Spell";
 
+    // Math
+    uint256 constant RAY  = 10 ** 27;
+
+    // --- Offboarding: Current Liquidation Ratio ---
+    uint256 constant CURRENT_UNI_A_MAT              =  165 * RAY / 100;
```
Comment (julienmartinlevrai) [inline src/Goerli-DssSpell.sol:43]: as per the spell sheet

```suggestion
    uint256 constant CURRENT_UNI_A_MAT              =  145 * RAY / 100;
```

File: src/Goerli-DssSpell.sol [hunk 2]
```diff
@@ -36,6 +36,23 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
...
+    uint256 constant CURRENT_UNIV2DAIETH_A_MAT      =  120 * RAY / 100;
+    uint256 constant CURRENT_UNIV2WBTCETH_A_MAT     =  145 * RAY / 100;
+    uint256 constant CURRENT_UNIV2UNIETH_A_MAT      =  160 * RAY / 100;
+    uint256 constant CURRENT_UNIV2WBTCDAI_A_MAT     =  120 * RAY / 100;
+
+    // --- Offboarding: Target Liquidation Ratio ---
+    uint256 constant TARGET_UNI_A_MAT               = 900 * RAY / 100;
```
Comment (julienmartinlevrai) [inline src/Goerli-DssSpell.sol:50]: as per the spell sheet

```suggestion
    uint256 constant TARGET_UNI_A_MAT               = 5_300 * RAY / 100;
```

File: src/Goerli-DssSpell.t.sol [hunk 3]
```diff
@@ -30,6 +30,26 @@ interface StarknetLike {
 
 contract DssSpellTest is GoerliDssSpellTestBase {
 
+    function testUNIALerpOffboardings() public {
```
Comment (julienmartinlevrai) [inline src/Goerli-DssSpell.t.sol:33]:
```suggestion
    function testUNIALerpOffboarding() public {
```

File: src/Goerli-DssSpell.t.sol [hunk 4]
```diff
@@ -30,6 +30,26 @@ interface StarknetLike {
...
+        checkIlkLerpOffboarding("UNI-A", "UNI-A Offboarding", 165, 900);
```
Comment (julienmartinlevrai) [inline src/Goerli-DssSpell.t.sol:34]:
```suggestion
        checkIlkLerpOffboarding("UNI-A", "UNI-A Offboarding", 145, 5300);
```

File: src/test/config.sol [hunk 5]
```diff
@@ -794,7 +794,7 @@ contract Config {
             clip_tail:    140 minutes,
             clip_cusp:    4000,
             clip_chip:    10,
-            clip_tip:     300,
+            clip_tip:     0,
```
Comment (julienmartinlevrai) [inline src/test/config.sol:797]: set `lerp` to `true` for all offboardings

**Review by julienmartinlevrai — CHANGES_REQUESTED** [review]
Please use `src/Goerli-DssSpellCollateralOnboarding.sol` instead of `src/Goerli-DssSpell.sol`

File: src/Goerli-DssSpell.sol (hunk 1 above)
Comment (nduchak) [reply src/Goerli-DssSpell.sol:43]: Are you sure about this one? These values are used for `UNIV2WBTCETH-A`. Also i check `spotter` and it has 165 for UNI-A

File: src/Goerli-DssSpell.t.sol (hunk 4 above)
Comment (lollike) [reply src/Goerli-DssSpell.t.sol:34]: Looks like there's a copy paste error in the implementation sheet. Correct values are found here as per the SR:
https://docs.google.com/spreadsheets/d/1miE1AOJsOXDXbcIDGvJkAefRChC7q_X713GvX00k_pM/edit#gid=1101084058

File: src/Goerli-DssSpell.sol (hunk 1 above)
Comment (lollike) [reply src/Goerli-DssSpell.sol]: Looks like there's a copy paste error in the implementation sheet. Correct values are found here as per the SR:
https://docs.google.com/spreadsheets/d/1miE1AOJsOXDXbcIDGvJkAefRChC7q_X713GvX00k_pM/edit#gid=1101084058

**Comment by hbarcelos** [comment]
> Please use `src/Goerli-DssSpellCollateralOnboarding.sol` instead of `src/Goerli-DssSpell.sol`

Maybe we should rename this file to `Goerli-DssSpellCollateral.sol` and have a separate method for off-boarding collateral.

**Review by julienmartinlevrai — APPROVED** [review]
* offboarding values match spell sheet and poll
* collaterals contract renamed correctly
* tests are correct and complete

LGTM

**Pull request merged**